### DISCUSSION TITLE
device compile TP-LINK

### DISCUSSION THREAD

**Discussion by mstrolo** [Ysurac/openmptcprouter#2224] [Q&A]
Hello all.

I want to build a version for my TP-link router.

model is TL-MR6400 V5

I have openWRT on it but how do i build a version for omr?
Do I need to build version one of the router itself?
can anyone help me with that?

**Comment by webmasterkai** [comment]
Unfortunately, I don't think your hardware is powerful enough to run OMR. The processor  is slow and 8MB of flash isn't enough to do much.

**Comment by ben-ba** [comment]
https://github.com/Ysurac/openmptcprouter/wiki/Create-image-for-unsupported-platform
